config/routes.rb

```diff
@@ -102,11 +102,17 @@ def matches?(request)
         get :per_week
       end
     end
+    resources :councillor_contributions, only:[:new, :create]
     collection do
       get :test_feed
     end
   end
 
+  post "/authorities/:authority_id/councillor_contributions/new", to: "councillor_contributions#new"
+
+  resources :contributors, only:[:new, :create]
+    get "/contributors/no_info", to: "contributors#no_contributor_info"
```

Comment:
I think this should change to something a little more specific like "contributors/anonymous", but we don't need to do that before deploying.